download.sh

```diff
@@ -0,0 +1,11 @@
+#!bin/bash
```

Comment:
Let's create a section in README.md .
How to run TinyLlama-1.1B-Chat

And please don't include this `download.sh`, it's too generic.

Reply:
good idea

Reply:
@tairov Could you have a look of the new readme file?